### ISSUE TITLE
Body scrolling should be disabled when modal is shown

### ISSUE THREAD

**Issue by tomivirkki** [vaadin/vaadin-overlay#19]
Consider preventing default for touchmove and wheel for the backdrop element

**Comment by jouni** [comment]
So, this would only apply to “globally modal” overlays, where there’s a visible backdrop? I mean, “locally modal” overlays/popups, like combo box and date picker, should allow scrolling, no? At least we allow that with the 1.x versions.

**Comment by tomivirkki** [comment]
Correct. This will only take place when there's backdrop/modality curtain present
